src/Components/FilterBar.jsx

```diff
@@ -1,23 +1,32 @@
-import React from 'react';
+import { useEffect, useState } from 'react';
+import { useFilter } from '../Context/filter-context';
+import axios from 'axios';
 function FilterBar() {
-	// const [catogData, setCatogData] = useState([]);
-	// const getCatog = async () => {
-	// 	const catog = await axios.get('/api/categories');
-	// 	try {
-	// 		console.log(catog.data.categories);
-	// 		setCatogData(catog.data.categories);
-	// 	} catch (err) {
-	// 		console.log('error');
-	// 	}
-	// };
-	// useEffect(() => {
-	// 	getCatog();
-	// }, []);
+	const { filter, filterDispatch } = useFilter();
+	const [categories, setCategories] = useState();
+	const getCategory = async () => {
+		const categories = await axios.get('/api/categories');
+
+		try {
+			console.log('catogs: ', categories.data.categories);
```

Comment:
remove consoles and use a code formatter tool  like preetier.

Reply:
I'll remove all of them once I'm done with the basic functionality